src/ppx/xr_component.cpp

```diff
@@ -53,6 +73,17 @@ void XrComponent::InitializeBeforeGrfxDeviceInit(const XrComponentCreateInfo& cr
 
     std::vector<const char*> xrInstanceExtensions;
     xrInstanceExtensions.push_back(graphicsAPIExtension);
+#if defined(PPX_ANDROID) and defined(PPX_OPENXR)
```

Comment:
This is needed for Quest builds. I think just checking for PPX_ANDROID should be enough.

Reply:
Same as the other, might be best not to enable it on normal android devices no?